user_docs/en/changes.md

```diff
@@ -27,6 +27,7 @@ To use this feature, "allow NVDA to control the volume of other applications" mu
 * Automatic language switching is now supported when using Microsoft Speech API version 5 (SAPI5) and Microsoft Speech Platform voices. (#17146, @gexgd0419)
 * NVDA can now be configured to speak the current line or paragraph when navigating with braille navigation keys. (#17053, @nvdaes)
 * In Word, the selection update is now reported when using Word commands to extend or reduce the selection (`f8` or `shift+f8`). (#3293, @CyrilleB79)
+* In Microsoft Word 16.0.18226 and higher, NVDA will now report if a heading is collapsed in both speech and braille. (#17499)
```

Comment:
No, the version number is still required, because people using a Word version lower than 16.0.18226 and higher or equal to 16.0.15000 on Windows 11 will have UIA enabled as default setting, but collapsed headings not reported.

```suggestion
* In Microsoft Word 16.0.18226 and higher or when using Word object model, NVDA will now report if a heading is collapsed in both speech and braille. (#17499)
```